source/ed/types.h

```diff
@@ -437,8 +437,13 @@ struct Poi {
     std::map<std::string, std::string> properties;
     std::string address_number = "";
     std::string address_name = "";
+    std::string uri = "poi:";
```

Comment:
minor: This prefix is a bit hidden elsewhere (OsmPoi, builder)... If you have an idea (naming, refactor, comment) to avoid futur problems (understanding, correcting for one bug, but breaking the other usecases...) it could be nice, but it's minor.